activation/poet.go

```diff
@@ -91,15 +105,37 @@ func defaultPoetClientFunc(target string) PoetProvingServiceClient {
 
 // NewHTTPPoetClient returns new instance of HTTPPoetClient for the specified target.
 func NewHTTPPoetClient(target string) *HTTPPoetClient {
-	return &HTTPPoetClient{baseURL: fmt.Sprintf("http://%s/v1", target)}
+	client := retryablehttp.NewClient()
+	client.RetryMax = 10
+	client.RetryWaitMin = time.Second
+	client.RetryWaitMax = time.Second * 2
```

Comment:
Should we make the retry waiting period configurable? For `systests` 1-2 seconds is probably fine but on mainnet I would make the spread bigger (maybe 1s - 1min) to avoid putting to much strain on a PoET when the whole network bombards them with requests ever 1-2 seconds.

Reply:
Yes, your suggestion sounds reasonable. Lets go with `CycleGap / 10` and `CycleGap / 100` 🙂